src/wallet/rpcdump.cpp

```diff
@@ -265,6 +297,33 @@ Value importwallet(const Array& params, bool fHelp)
         boost::split(vstr, line, boost::is_any_of(" "));
         if (vstr.size() < 2)
             continue;
+
+        // Let's see if the address is a valid Zcash spending key
+        if (fImportZKeys) {
+            try {
+                CZCSpendingKey spendingkey(vstr[0]);
+                libzcash::SpendingKey key = spendingkey.Get();
+                libzcash::PaymentAddress addr = key.address();
+                if (pwalletMain->HaveSpendingKey(addr)) {
+                    LogPrintf("Skipping import of zaddr %s (key already present)\n", CZCPaymentAddress(addr).ToString());
+                    continue;
+                }
+                int64_t nTime = DecodeDumpTime(vstr[1]);
+                LogPrintf("Importing zaddr %s...\n", CZCPaymentAddress(addr).ToString());
+                if (!pwalletMain->AddZKey(key)) {
+                    // Something went wrong
+                    fGood = false;
+                    continue;
```

Comment:
I'll file a ticket about this so we don't block release.  The byte prefixes for mainnet and testnet are defined internally in chainparams.cpp and are not exposed.  If we hard-code a string comparison we might forget to change it if the prefix were to ever change. Ideally we should call a utility function somewhere, perhaps on chainparams, which returns a bool e.g. looksLikeSpendingKey() or looksLikePaymentAddress().